changelog.d/test-django52.md

```diff
@@ -0,0 +1 @@
+Set up and test on Django 5.2, and make it possible to test on Python 3.13.
```

Comment:
This makes it sounds like we're not yet testing on Python 3.13, only add the capability